ovs/extensions/healthcheck/volumedriver/volumedriver_health_check.py

```diff
@@ -128,30 +132,49 @@ def check_volumedrivers(logger):
                 name = "ovs-healthcheck-test-{0}.raw".format(VolumedriverHealthCheck.MACHINE_ID)
                 if vp.guid in VolumedriverHealthCheck.MACHINE_DETAILS.vpools_guids:
                     try:
-                        file_path = "/mnt/{0}/{1}".format(vp.name, name)
-                        vdisk_guid = VolumedriverHealthCheck._check_volumedriver(name, vp.guid)
-                        time.sleep(VolumedriverHealthCheck.VDISK_TIMEOUT_BEFORE_DELETE)
-                        if os.path.exists(file_path):
-                            # working
-                            VolumedriverHealthCheck._check_volumedriver_remove(vdisk_guid)
-                            logger.success("Volumedriver of vPool '{0}' is working fine!".format(vp.name),
-                                           'volumedriver_{0}'.format(vp.name))
+                        # delete if previous vdisk with this name exists
+                        if VolumedriverHealthCheck._check_volumedriver_remove(vpool_name=vp.name, vdisk_name=name,
```

Comment:
Why do you expect there to be a vdisk already present with the name?
This points out that the healthcheck can't cleanup decently after a previous failed run.